.gitignore

```diff
@@ -12,5 +12,7 @@ venv/
 build/
 dist/
 
-data/
+data/*
+!data/
+!data/README.md
```

Comment:
why are these added? keep in mind the data folder is not usually in the repo it's oten in `~/archivebox` or somewhere else entirely on the system.